src/modules/drive/Toolbar/components/ShortcutCreationModal.spec.jsx

```diff
@@ -43,11 +46,12 @@ describe('ShortcutCreationModal', () => {
     jest.resetAllMocks()
     // TODO: cozy-ui  Warning: Failed prop type: The prop `open` is marked as required in `FixedDialog`, but its value is `undefined`.
     // TODO: cozy-ui  Material-UI: You are trying to override a style that does not exist - Fix the `borderWidth` key of `theme.overrides.MuiTextField`
-    jest.spyOn(console, 'error').mockImplementation()
-    jest.spyOn(console, 'warn').mockImplementation()
+    // jest.spyOn(console, 'error').mockImplementation()
```

Comment:
If the test works without the spyOn, can we remove them instead of comment them?

Reply:
oops c'est un oubli